tasks/src/test/java/webdriver/google_cloud/page/GoogleCloudCalculatorPage.java

```diff
@@ -4,50 +4,57 @@
 import static org.openqa.selenium.Keys.ENTER;
 import static webdriver.google_cloud.browser.Browser.*;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 
 public class GoogleCloudCalculatorPage extends GoogleCloudAbstractPage {
-  private WebElement webElement;
+  private final Logger logger = LogManager.getRootLogger();
 
   @FindBy(xpath = "//div[@title='Compute Engine']/..")
   private WebElement computeEngineButton;
-  @FindBy(id = "input_53")
+  @FindBy(id = "input_55")
```

Comment:
It looks like your ids are automatically generated. Probably you can use another more appropriate strategy for searching such web elements?